testing/wcag/.pa11yci.local.json

```diff
@@ -10,8 +10,6 @@
     "http://localhost:6006/iframe.html?id=2-design-foundations--colour-language",
     "http://localhost:6006/iframe.html?id=2-design-foundations--colour-palette",
     "http://localhost:6006/iframe.html?id=3-components--advice-feedback",
-    "http://localhost:6006/iframe.html?id=3-components--advice-feedback",
```

Comment:
Backstop often uses the same URL for multiple scenarios. I updated the pa11y script to convert the urls to a `Set` first to de-dupe this list. Avoids testing the same URL multiple times.